src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -98,4 +108,36 @@ public DarCollection updateDarCollection(Integer collectionId, User user) {
     darCollectionDAO.updateDarCollection(collectionId, user.getDacUserId(), new Date());
     return getByCollectionId(collectionId);
   }
+
+  public void addDatasetsToCollections(List<DarCollection> collections) {
+    
+    List<List<DataAccessRequest>> collectionDars = collections.stream()
+      .map(c -> c.getDars())
```

Comment:
This can be simplified to
```suggestion
      .map(DarCollection::getDars)
```